kubernetes/sequencer/scripts/start-relayer.sh

```diff
@@ -14,8 +14,6 @@ fi
 
 echo "Celestia Bearer token fetched successfully."
 
-echo "celestia_bearer_token = \"$BEARER_TOKEN\"" > "$home_dir"/SequencerRelayerConfig.toml
```

Comment:
Is this just cleanup?

Reply:
Yeah. We don't need the config toml because we are exporting the value to an envar.